src/full/Agda/TypeChecking/Monad/Builtin.hs

```diff
@@ -78,10 +76,6 @@ litType = \case
   where
     el t = El (mkType 0) t
 
-instance MonadIO m => HasBuiltins (TCMT m) where
-  getBuiltinThing b = liftM2 mplus (Map.lookup b <$> useTC stLocalBuiltins)
-                      (Map.lookup b <$> useTC stImportedBuiltins)
```

Comment:
Not altered here, but I wonder why this needs a `MonadIO` constraint.

Reply:
I think `TCMT` has `MonadIO` as a constraint on its argument?